contracts/src/BeefyClient.sol

```diff
@@ -448,22 +448,10 @@ contract BeefyClient {
     ) internal pure returns (uint256) {
         // Start with the minimum number of signatures.
         uint256 numRequiredSignatures = minRequiredSignatures;
-
-        // We must subtract minimumSignatures from the number of validators or we might end up
-        // requiring more signatures than there are validators.
-        uint256 extraValidatorsLen = validatorSetLen.saturatingSub(minRequiredSignatures);
-        if (extraValidatorsLen > 1) {
-            numRequiredSignatures += Math.log2(extraValidatorsLen, Math.Rounding.Ceil);
```

Comment:
We do not need to check `extraValidatorsLen` as `Math.log2(extraValidatorsLen, Math.Rounding.Ceil)` returns `0` with `0` as input.